x-pack/plugins/upgrade_assistant/server/routes/cluster_checkup.ts

```diff
@@ -13,8 +13,6 @@ import { reindexActionsFactory } from '../lib/reindexing/reindex_actions';
 import { reindexServiceFactory } from '../lib/reindexing';
 
 export function registerClusterCheckupRoutes({ cloud, router, licensing, log }: RouteDependencies) {
```

Comment:
I think `cloud` dependency is not needed anymore, it can be even removed from server side dependencies altogether.

Reply:
Good catch! I've removed the `cloud` dependency.